src/main/kotlin/com/karumi/maxibonkata/Developer.kt

```diff
@@ -0,0 +1,14 @@
+package com.karumi.maxibonkata
+
+data class Developer internal constructor(val name: String = "", private val numberOfMaxibons: Int = 0) {
```

Comment:
Developer seems to be a domain object. Are you sure you don't want to allow the creation of new developers?

Reply:
Yeah, `Developer` is a domain object but we only have N possible developers (the ones defined in the object aka The Karumis). I think an enum would better fit this type but in order to test future developers we are doing it in this way. We can create new developers on the fly and testing that the whole system works while (soft) forcing the production code to only be able to create Karumis.